platform/darwin/src/local_glyph_rasterizer.mm

```diff
@@ -96,14 +122,30 @@ CTFontRef getFont() {
 {}
 
 bool LocalGlyphRasterizer::canRasterizeGlyph(const FontStack&, GlyphID glyphID) {
-    return util::i18n::allowsFixedWidthGlyphGeneration(glyphID) && impl->getFont();
+    return util::i18n::allowsFixedWidthGlyphGeneration(glyphID) && impl->isEnabled();
```

Comment:
This diff is a bit hard to follow. Before, glyphs within the limited CJK ranges would be rasterized locally unless no local font family was specified (disabling the feature). In the case where `getFont()` failed to create a `CTFontRef`, it would throw an exception, not return `false`. Moreover, that case would never occur, because `CTFontCreateWithFontDescriptor()` always returns a best-match font (Helvetica as a last resort fallback).

The new code makes the disabled case more explicit.